src/libraries/System.Net.Quic/src/System/Net/Quic/QuicError.cs

```diff
@@ -72,5 +72,10 @@ public enum QuicError
         /// The operation has been aborted.
         /// </summary>
         OperationAborted,
+
+        /// <summary>
+        /// OS does not support IPv6
+        /// </summary>
+        OSDoesNotSupportIPv6
```

Comment:
This needs to go through API review, unless done already. It's the same thing as with https://github.com/dotnet/runtime/issues/74755